number.py

```diff
@@ -1,28 +1,4 @@
 # Import the math module using import keyword.
 import math
```

Comment:
This is a Unusedimportmath
```suggestion 
```

 from our GitHub Enhacement Bot